From fbba14e2e85f11b7223fb922a3b26a94931239b4 Mon Sep 17 00:00:00 2001
From: kfx <kfx@rock-chips.com>
Date: Tue, 9 Jul 2013 10:55:13 +0800
Subject: [PATCH] fixed compile error: drivers/mmc/card/block.c:992:46: error:
 request for member 'stop' in something not a structure or union

---
 drivers/mmc/card/block.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/card/block.c b/drivers/mmc/card/block.c
index aa0b48116ba8..ff810129feb2 100644
--- a/drivers/mmc/card/block.c
+++ b/drivers/mmc/card/block.c
@@ -980,7 +980,7 @@ static int sdmmc_blk_issue_rw_rq(struct mmc_queue *mq, struct request *req)
 		 * stop.error indicates a problem with the stop command.  Data
 		 * may have been transferred, or may still be transferring.
 		 */
-		if (brq->sbc.error || brq->cmd.error || brq.stop.error) {
+		if (brq->sbc.error || brq->cmd.error || brq->stop.error) {
 			switch (mmc_blk_cmd_recovery(card, req, brq)) {
 			case ERR_RETRY:
 				if (retry++ < 5)
-- 
2.35.3

